packages/frint-router-react/src/Route.js

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getMountableComponent } from 'frint-react';
+/* eslint-enable import/no-extraneous-dependencies */
+
+export default class Route extends React.Component {
+  static contextTypes = {
+    app: PropTypes.object.isRequired
+  };
+
+  static propTypes = {
+    path: PropTypes.string,
+    exact: PropTypes.bool,
+    computedMatch: PropTypes.object,
+    component: PropTypes.oneOfType([PropTypes.node, PropTypes.func]),
```

Comment:
here we expect a definition of component, a class, not an instance (`Component`, not `<Component />`). that's why `PropTypes.element` doesn't fit here. so it's a function, because JavaScript :). also there can be stateless components, those are definitely functions.
actually `PropTypes.node` might not be needed. I am going to check.